samples/gltf_viewer.cpp

```diff
@@ -402,8 +431,8 @@ static void colorGradingUI(App& app) {
         colorGrading.toneMapping = (decltype(colorGrading.toneMapping)) toneMapping;
 
         if (ImGui::CollapsingHeader("White balance")) {
-            ImGui::SliderInt("Temperature", &colorGrading.temperature, -100, 100);
-            ImGui::SliderInt("Tint", &colorGrading.tint, -100, 100);
+            ImGui::SliderFloat("Temperature", &colorGrading.temperature, -1.0f, 1.0f);
```

Comment:
The actual API takes a float between -1 and 1, and the `Settings` struct is 1:1 with the API, so I think this is a bit more straightforward.

Reply:
The UI uses -100/+100 because that's what tools like Photoshop/Lightroom/etc. expose.  There are other settings that I think we need to expose differently in the UI than in our API (for instance angles should be in degrees in the UI, not in radians).

Reply:
Ah, got it.  I'll fix this using temporary integers.

Reply:
Fixed.